Hello/about.html

```diff
@@ -0,0 +1,68 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <title>Об авторе</title>
+  </head>
+  <body>
+    <h1>Приветствую!</h1>
+    <h3>Почему я выбрала Front-end?</h3>
+    <p>
+      Я люблю творчество и логические задачки. Иногда нестандартно нахожу
+      решение. Желание заняться программированием возникло с появлением
+      удаленного формата работы, выходом в декретный отпуск.
+      <br />Немаловажна привлекающая заработная плата разработчиков &#x1F63A;.
+      Ну а наиболее творческое направление я увидела у
+      <em>Front-end</em> разработчиков. Сюда и направилась.
+    </p>
+    <h3>Напишу немного о себе</h3>
+    <p>
+      Родилась я 29 лет назад. Школа, медицинский университет, интернатура,
+      работа. На данный момнет я работаю в
+      <del>поликлинике</del>
+      <abbr title="Гомельская областная инфекционная клиническая больница"
+        >ГОИКБ</abbr
```

Comment:
Hereinafter tags shouldn't be visibly broken.
Check the prettier's settings, better set it to default.

Reply:
repeated attempts to correct the error did not give results.